app/src/main/java/com/babylon/wallet/android/presentation/status/transaction/TransactionStatusDialogNav.kt

```diff
@@ -63,12 +69,26 @@ fun NavController.transactionStatusDialog(transactionEvent: AppEvent.Status.Tran
         null
     }
 
+    val txProcessingTime = if (transactionEvent is AppEvent.Status.Transaction.Fail) {
+        Json.encodeToString(transactionEvent.txProcessingTime)
+    } else {
+        null
+    }
+
+    val walletErrorType = if (transactionEvent is AppEvent.Status.Transaction.Fail) {
+        Json.encodeToString(transactionEvent.walletErrorType)
+    } else {
+        null
+    }
+
     val requestId = transactionEvent.requestId.ifBlank { error("Transaction id cannot be empty") }
     navigate(
         route = "$ROUTE/${transactionEvent.toType()}/$requestId" +
```

Comment:
maybe we could define custom argument type for this route, like `RequiredPersonaFieldsParameterType` to avoid passing so many arguments?